src/org/omegat/core/team2/RemoteRepositoryProvider.java

```diff
@@ -480,6 +536,37 @@ public String getVersion() throws Exception {
             return repo.getFileVersion(new File(repoMapping.getRepository(), filterPrefix).getPath());
         }
 
+
+        /**
+         * Specialized version of copy() to copy omegat.project in remote repository to
+         * omegat.project.NEW
+         *
+         * @return Relative paths of copied files, <em>with <code>/</code> at
+         *         start and end</em>
+         */
+        protected List<String> copyTempNew(File from, File to, String prefix, List<String> includes,
+                                    List<String> excludes, String eolConversionCharset) throws IOException {
```

Comment:
We can change this signature to 

```suggestion
        protected List<String> copy(File from, File to, String prefix, String postfix, List<String> includes,
                                    List<String> excludes, String eolConversionCharset) throws IOException {
```

and original `copy` can be
```java
        protected List<String> copy(File from, File to, String prefix,  List<String> includes,
                                    List<String> excludes, String eolConversionCharset) throws IOException {
            return copy(from, to, prefix, "", includes, excludes, eolConversionCharset);
        }
```

the caller can be
```java
    copy(from, to, "", ".NEW", includes, excludes, null);
```